### ISSUE TITLE
If a goroutine panics, the qps is reported as 0

### ISSUE THREAD

**Issue by sandeeprenjith** [sandeeprenjith/dnsblast#13]
```
[tester@fc3t-vm builds]$ ./dnsblast -s 192.168.130.9 -r 100000 -t 10 -l 10

EXECUTING TEST
+-----------------------------------------------------------+
2020/03/14 01:16:22 QPS/Thread:  32  Latency:  30.119956ms
2020/03/14 01:16:23 QPS/Thread:  67  Latency:  29.23337ms
2020/03/14 01:16:24 QPS/Thread:  104  Latency:  28.351538ms
2020/03/14 01:16:25 QPS/Thread:  71  Latency:  54.71489ms
2020/03/14 01:16:26 QPS/Thread:  173  Latency:  28.661596ms
2020/03/14 01:16:27 QPS/Thread:  208  Latency:  28.520234ms
2020/03/14 01:16:28 QPS/Thread:  205  Latency:  33.697322ms
2020/03/14 01:16:29 QPS/Thread:  263  Latency:  30.143987ms
2020/03/14 01:16:30 QPS/Thread:  32  Latency:  270.394814ms
2020/03/14 01:16:31 QPS/Thread:  226  Latency:  43.702729ms
Looks like the program paniced.
This usually happens due issues with server responses.(If you want to debug)
2020/03/14 01:16:32 QPS/Thread:  127  Latency:  85.197041ms
2020/03/14 01:16:33 QPS/Thread:  216  Latency:  54.86297ms
2020/03/14 01:16:34 QPS/Thread:  428  Latency:  30.096478ms
+-----------------------------------------------------------+

  REPORT
+---------------------+------------------------+
| Target Server       | udp://192.168.130.9:53 |
| Test                | Uncached Responses     |
| Send Rate           | 100000 Queries/Sec     |
| Threads             | 10                     |
| Duration of test    | 10 Sec                 |
| Protocol            | UDP                    |
| Average Queries/Sec | 0                      |
| Average Latency     | 0s                     |
+---------------------+------------------------+
```

**Comment by sandeeprenjith** [comment]
Fixed with commit aa5927abddd172202ac3fbbafb12fa864e6029ee

**Comment by sandeeprenjith** [comment]
```
$ ./dnsblast -s 192.168.130.9 -r 100000 -t 10 -l 10

EXECUTING TEST
+-----------------------------------------------------------+
2020/03/14 02:56:57 QPS/Thread:  1181  Latency:  584.508µs
2020/03/14 02:56:58 QPS/Thread:  2330  Latency:  607.524µs
2020/03/14 02:56:59 QPS/Thread:  3481  Latency:  607.024µs
2020/03/14 02:57:00 QPS/Thread:  4634  Latency:  609.79µs
2020/03/14 02:57:01 QPS/Thread:  5719  Latency:  615.258µs
2020/03/14 02:57:02 QPS/Thread:  2954  Latency:  1.44521ms
2020/03/14 02:57:03 QPS/Thread:  773  Latency:  6.435901ms
2020/03/14 02:57:04 QPS/Thread:  3329  Latency:  1.714003ms
2020/03/14 02:57:05 QPS/Thread:  7516  Latency:  857.504µs
2020/03/14 02:57:06 QPS/Thread:  9380  Latency:  762.656µs
A thread ran into an issue. Consider reducing the number of threads
2020/03/14 02:57:06 QPS/Thread:  10000  Latency:  763.846µs
2020/03/14 02:57:06 QPS/Thread:  10000  Latency:  770.091µs
2020/03/14 02:57:06 QPS/Thread:  10000  Latency:  769.701µs
2020/03/14 02:57:06 QPS/Thread:  10000  Latency:  768.923µs
2020/03/14 02:57:07 QPS/Thread:  223  Latency:  35.283075ms
2020/03/14 02:57:08 QPS/Thread:  805  Latency:  10.700064ms
2020/03/14 02:57:09 QPS/Thread:  1796  Latency:  5.179659ms
2020/03/14 02:57:09 QPS/Thread:  10000  Latency:  975.269µs
2020/03/14 02:57:10 QPS/Thread:  4731  Latency:  2.121817ms
+-----------------------------------------------------------+

  REPORT
+---------------------+------------------------+
| Target Server       | udp://192.168.130.9:53 |
| Test                | Uncached Responses     |
| Send Rate           | 100000 Queries/Sec     |
| Threads             | 10                     |
| Duration of test    | 10 Sec                 |
| Protocol            | UDP                    |
| Average Queries/Sec | 9380                   |
| Average Latency     | 762.656µs              |
+---------------------+------------------------+
```
